src/samarium/core.py

```diff
@@ -97,16 +98,16 @@ def import_module(data: str, reg: Registry) -> None:
                 )
 
 
-def print_safe(*args) -> Any:
-    args = list(map(correct_type, args))
-    return_args = args[:]
-    args = list(map(str, args))
-    types = [*map(type, args)]
+def print_safe(*args: Attrs | bool | None) -> Any:
```

Comment:
Pretty sure it always returns `Attrs` (considering `bool` and `None` get converted to a Samarium type)

Reply:
```suggestion
def print_safe(*args: Attrs | bool | None) -> Attrs:
```